common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -176,6 +198,18 @@
                 this.updatePlayTime(time);
             },
 
+            handleCaptionToggle: function(event) {
+                var KEY = $.ui.keyCode,
+                    keyCode = event.keyCode;
+
+                switch(keyCode) {
+                    // KEY.ENTER works inherently so we don't need to script it explicitly
```

Comment:
Does `KEY.ENTER` work inherently because it is perceived as a "click" event, which is handled already?  Or is there something else I'm missing about how this works?